### PR TITLE
[elects] Bump to version 0.6.1

### PR DESCRIPTION
**Pull request by dagurval**
Bump electrs to 0.6.1

I also added a basic reorg test. This test would triggering an assert in electrs and thus fail with previous version.

[Full change log](https://github.com/romanz/electrs/blob/master/RELEASE-NOTES.md)

### CODE REVIEW

**Comment by dagurval** [comment]
Unit tests updated

File: qa/rpc-tests/test_framework/electrumutil.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright (c) 2015-2017 The Bitcoin Unlimited developers
```
Comment (sickpig) [inline qa/rpc-tests/test_framework/electrumutil.py]: wrong copyright year

File: qa/rpc-tests/electrum_reorg.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# Copyright (c) 2015-2017 The Bitcoin Unlimited developers
```
Comment (sickpig) [inline qa/rpc-tests/electrum_reorg.py]: wrong copyright year

**Comment by sickpig** [comment]
@dagurval a part from the copyright years nit I think this is good to go. Will approve as soon as you fix the nit :smile:

File: qa/pull-tester/rpc-tests.py [hunk 3]
```diff
@@ -245,7 +245,8 @@ def option_passed(option_without_dashes):
     'sighashmatch',
     'getlogcategories',
     'getrawtransaction',
-    Disabled('electrum_basics', "Needs to be skipped if electrs is not built")
+    Disabled('electrum_basics', "Needs to be skipped if electrs is not built"),
+    Disabled('electrum_reorg', "Needs to be skipped if electrs is not built")
```
Comment (ptschip) [inline qa/pull-tester/rpc-tests.py:249]: Can we add the same method we use for the zmq tests (see ENABLE_ZMQ in rpc-tests.py)...so we would create an ENABLE_ELECTR variable when we build and that would get picked up in the python ./rpc-tests.py when it runs.

**Comment by dagurval** [comment]
copyright headers fixed

File: qa/pull-tester/rpc-tests.py (hunk 3 above)
Comment (dagurval) [reply qa/pull-tester/rpc-tests.py:249]: That's a good idea (for another PR)

**Review by sickpig — APPROVED** [review]

**Pull request merged**